### ISSUE TITLE
Person component enhancement

### ISSUE THREAD

**Issue by MichaelZanetti** [qJake/HADotNet.CommandCenter#4]
Although you can use the `state tile` it would probably make sense to allow the `person tile` to use the `person` entity from Home Assistant instead of only `device_tracker`.

**Comment by qJake** [comment]
@MichaelZanetti Makes sense - I don't have any `person` entities in my setup yet (they are still device trackers - I need to update! 😁)

Can you confirm if the `person` entity for a phone still has attributes for `friendly_name` and `entity_picture`, and that its `state` text is the location (i.e. "home", "work", "not-home")? If so, it's a very easy change to also include `person` device types!

**Comment by MichaelZanetti** [comment]
Hi @qJake , yes the `person` entity still has state text like "home", "not_home".
Here's the attributes mine has!

```
editable: false
id: id_michael
source: device_tracker.iphonexs_michael
user_id: <id>
friendly_name: Michael
entity_picture: /local/Michael.jpg
```

**Comment by qJake** [comment]
Perfect, thanks! I've fixed this. It will be released in the next version (0.4.0) soon!

**Comment by qJake** [comment]
[Released!](https://github.com/qJake/HADotNet.CommandCenter/releases/tag/v0.4.0)
